.gitattributes

```diff
@@ -4,3 +4,5 @@ Cargo.lock merge=binary
 
 # Use unix line endings everywhere, even on Windows
 * text=auto eol=lf
+services/mgmt/**/*.rs linguist-generated=true
+services/mgmt/**/Cargo.toml linguist-generated=true
```

Comment:
This makes it so GitHub knows these files are generated and does not display them in the diffs by default. @heaths requested this. It is working.